packages/calypso-products/src/features-list.tsx

```diff
@@ -760,7 +767,10 @@ const FEATURES_LIST: FeatureList = {
 
 	[ FEATURE_VIDEO_UPLOADS ]: {
 		getSlug: () => FEATURE_VIDEO_UPLOADS,
-		getTitle: () => i18n.translate( 'VideoPress support' ),
+		getTitle: () =>
+			getTrailMapExperiment()
+				? i18n.translate( 'Upload 4K video with VideoPress' )
+				: i18n.translate( 'VideoPress support' ),
```

Comment:
Can you clarify why `getTrailMapExperiment` is used here, as opposed to `isTrailMapVariant` which seems to be used is other cases?